SA3/qa_web/templates/qa_web/EditUserProfile.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Edit User Profile</title>
+    </br>
+    <h1> Account Setting</h1>
+
+</head>
+<body>
+
+    <form>
```

Comment:
I feel like this form can be generated through Django Forms, to be easily handled in the backend.